webhook/models.py

```diff
@@ -1,9 +1,23 @@
 from django.db import models
+import json
+
+IFTTT = 'IT'
+HIPCHAT = 'HC'
+UNKNOWN = 'UK'
+SOURCE_CHOICES = (
+    (IFTTT, 'If This Then That'),
+    (HIPCHAT, 'Hipchat'),
+    (UNKNOWN, 'Unknown'),
+)
 
 class IncomingRequest(models.Model):
     last_edited = models.DateTimeField(auto_now=True,db_index=True)
-    created = models.DateTimeField(auto_now_add=True,db_index=True)
-    source = models.CharField(blank=True, null=True, max_length=100)
+    created = models.DateTimeField(auto_now_add=True,db_index=True)    
+    source = models.CharField(max_length=2, choices=SOURCE_CHOICES, default=UNKNOWN)
     incoming_url = models.URLField(blank=True, null=True)
     payload = models.TextField(default='{}')
-    user = models.IntegerField()
+    user = models.IntegerField(default='-1')
+
+    @property
```

Comment:
You can use the `@property` decorator to make a function act like a field. For example, you can now do: 

```
icr = IncomingRequest()
icr.payload_as_json
```

(see: `IncomingRequestTests.test_payload_as_json`)